docs/docs/project.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: docs
+title: Project API
+permalink: project
+---
+
+# Project API
+
+Github4s supports the [Project API](https://developer.github.com/v3/projects/). As a result,
+with Github4s, you can interact with:
+
+- [Projet](#project)
+  - [List projects](#list-project)
+  - [Columns](#columns)
+    - [List project columns](#list-projects-columns)
+
+The following examples assume the following imports and token:
+
+```scala mdoc:silent
+import github4s.Github
+import github4s.GithubIOSyntax._
+import cats.effect.IO
+import scala.concurrent.ExecutionContext.Implicits.global
+
+implicit val IOContextShift = IO.contextShift(global)
+val accessToken = sys.env.get("GITHUB4S_ACCESS_TOKEN")
+```
+
+They also make use of `cats.Id`, but any type container `F` implementing `ConcurrentEffect` will do.
+
+LiftIO syntax for `cats.Id` and `Future` are provided in `GithubIOSyntax`.
+
+## Project
+
+### List projects
+
+Note: The Projects API is currently available for developers to preview. During the preview period,
+the API may change without advance notice. Please see the blog post for full details. To access the
+API during the preview period, you must provide a custom media type in the `Accept` header:
+ `application/vnd.github.inertia-preview+json`
+
+You can list the project for a particular organization with `listProjects`; it takes as arguments:
+
+- `org`: name of the organization for which we want to retrieve the projects.
+- `state`: filter projects returned by their state. Can be either `open`, `closed`, `all`. Default: `open`, optional
+- `pagination`: Limit and Offset for pagination, optional.
+- `header`: headers to include in the request, optional.
+
+
+
+To list the projects for organization `47deg`:
+
+```scala mdoc:compile-only
+val listProjects = Github[IO](accessToken).projects.listProjects(org = "47deg",headers = Map("Accept" -> "application/vnd.github.inertia-preview+json"))
```

Comment:
:art: it might make sense to break that line into 2, so that it renders better on the website